src/Theme/Registry.php

```diff
@@ -39,4 +40,29 @@
             }
         }
     }
+
+    protected function postProcessExtension(array &$cache, ActiveTheme $theme)
```

Comment:
Move to its own PR, test with a module which has implemented `hook_process` to modify output. See if Examples module has something